ngageoint/opensphere
# feat(csvImport): Add option to pick icon as part of CSV import

When importing a file, the options step allows selection of an initial color - this adds the choice of selecting an icon as well

![Screenshot from 2019-05-21 11-23-16](https://user-images.githubusercontent.com/35077650/58117360-a0e56600-7bbb-11e9-9b42-18cb3f78a628.png)

Live demo: http://csv-icon-branch-opensphere-ngageoint.surge.sh/
To test it out:
1. Import a CSV file and in the last step pick a shape such as an icon
2. Verify that the created layer starts off with it's features showing that shape
3. Import another CSV file and verify that not selecting an icon defaults the features to be a point
4. Repeat steps 1-3 with several other file types - shp, geojson, gml, gpx, kml, and georss should all be supported
